packages/devtools_app/test/association_variable_test.dart

```diff
@@ -22,10 +19,11 @@ final libraryRef = LibraryRef(
 );
 
 void main() {
-  ServiceConnectionManager manager;
+  ServiceConnectionManager? manager;
```

Comment:
make this late and non-nullable and we can get rid of all the null asserts on `manager` later